README.md

```diff
@@ -33,8 +33,8 @@ The recommended way to install Semantic MediaWiki is by using [Composer][compose
 
 Most of the documentation can be found on the [Semantic MediaWiki wiki](https://www.semantic-mediawiki.org).
 A small core of documentation also comes bundled with the software itself. This documentation is minimalistic
-and less explanatory than what can be found on the SMW wiki. It is however always kept up to date, and applies
-to the version of the code it comes bundled with. The most important files are linked below.
+and less explanatory than what can be found on the SMW wiki. However, It is always kept up to date and applies
+to the version of the code it bundles with. The most critical files are linked below.
```

Comment:
critical might score higher for "engagement", but it's less clear, so I think the wrong choice for docs in a readme